services/121-service/src/registration/distinct-regisration-pair.entity.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Column,
+  Entity,
+  Index,
+  JoinColumn,
+  ManyToOne,
+  Relation,
+  Unique,
+} from 'typeorm';
+
+import { Base121Entity } from '@121-service/src/base.entity';
+import { RegistrationEntity } from '@121-service/src/registration/registration.entity';
+
+@Entity('distinct_registration_pair')
+@Unique(['smallerRegistrationId', 'largerRegistrationId'])
```

Comment:
Its actually "lower" v.s. "higher", I'd think...
As these IDs are 'ordered on a line', not "things you can measure/weigh on a scale"...

But I don't think @RubenGeo would like to open up these cans-of-naming-worms... ;)

Reply:
resolved with rebase